app/views/case_studies/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for :title, "#{@content_item.page_title} - #{I18n.t("formats.#{@content_item.document_type}", count: 1)} - GOV.UK" %>
+
+<% content_for :extra_headers do %>
+  <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :article } %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds responsive-top-margin">
+    <%= render 'govuk_publishing_components/components/title', @case_study.title_and_context %>
+  </div>
+  <%= render 'shared/translations' %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+  </div>
+</div>
+
```

Comment:
You might need to look at the take part PR around adding a main class here (this might also affect help pages, FYI)